examples/08-averaging/01-average_across_bodies.py

```diff
@@ -44,17 +49,16 @@
 meshes.plot(text="Body meshes")
 
 ###############################################################################
-# As you can see in the preceding image, even though the piston rod is one single part,
-# it is composed of two different bodies. Additionally, you can see that the region
-# where the two bodies are bonded together contains nodes that are common between them.
+# As can be seen in the preceding image, even though the piston rod is one single part,
+# it is composed of two different bodies. Additionally, their interface share common nodes.
```

Comment:
I can't tell from the sentence if there is more than one interface or just one. 

One interface: Additionally, their interface shares common nodes.
More than one: Additionally, their interfaces share common nodes.